source/urban_cooling_model.rst

```diff
@@ -159,104 +158,103 @@ For each pixel, the model computes the estimated loss in productivity, in %, for
+    * Green_area: Required. A value of either 0 or 1, 1 meaning that the LULC class qualifies as a green area (green areas >2ha have an additional cooling effect), and 0 meaning that the class is not counted as a green area.
+    * Building_intensity: A floating-point value between 0 and 1. This is calculated by dividing the floor area by the land area, normalized between 0 and 1. Required if using the weighted factor approach to CC calculations.
 
-* Reference evapotranspiration: a raster representing reference evapotranspiration (units of millimeters) for the period of interest (could be a specific date or monthly values can be used as a proxy)
+* Reference Evapotranspiration: A raster representing reference evapotranspiration (units of millimeters) for the period of interest (could be a specific date or monthly values can be used as a proxy).
 
-* Areas of interest: polygon vector delineating areas of interest (city boundaries or neighborhoods boundaries). Results will be aggregated within each shape contained in this vector
+* Area of interest: Polygon vector delineating areas of interest (city boundaries or neighborhoods boundaries). Results will be aggregated within each shape contained in this vector.
 
-* Green Area Maximum Cooling Distance (:math:`d_{cool}`) : Distance (in meters) over which large urban parks (> 2 ha) will have a cooling effect
+* Green Area Maximum Cooling Distance (:math:`d_{cool}`): Distance (in meters) over which large urban parks (>2ha) will have a cooling effect (default value: 450m).
 
-* Reference Air Temperature (:math:`T_{ref}`): Rural reference temperature (where the urban heat island effect is not observed) for the period of interest. This could be nighttime or daytime temperature, for a specific date or an average over several days. The results will be given for the same period of interest).
+* Baseline air temperature (:math:`T_{ref}`): Rural reference air temperature (where the urban heat island effect is not observed) for the period of interest. This could be nighttime or daytime temperature, for a specific date or an average over several days. The results will be given for the same period of interest.
 
-* Magnitude of the UHI Effect (:math:`UHI_{max}`) : Magnitude of the urban heat island effect, in degrees Celcius, i.e. the difference between the rural reference temperature and the maximum temperature observed in the city.
+* Magnitude of the UHI effect (:math:`UHI_{max}`): Magnitude of the UHI effect (in degC), i.e. the difference between the rural reference (baseline air) temperature and the maximum temperature observed in the city.
 
-* Air Temperature Maximum Blending Distance: Search radius (in meters) used in the moving average to account for air mixing (default value: 2000m)
+* Air Temperature Maximum Blending Distance: Search radius (in meters) used in the moving average to account for air mixing (default value: 600m).

```

Comment:
The UI does set a default of 2000 for this input [here](https://github.com/natcap/invest/blob/01a3ddeac94d60afd124542b15503fdf916abb0f/src/natcap/invest/ui/urban_cooling_model.py#L75) .However, I'm not sure about the concept of a "default" value to begin with. Here are some problems I see with it: 
- When you save the inputs from the last model run, any defaults are overridden. As far as I know there isn't a way to "reset defaults" in the current UI.
- This is because a "default" is handled entirely on the UI side: it's simply a string that's put into the input field when it first loads.
- There is no representation of a "default value" in the `ARGS_SPEC`, and I'm not convinced that there should be.
- Because it's not represented in the `ARGS_SPEC`, it will not carry over to the workbench UI.
- All a "default" really means is a commonly-accepted value that may be suitable for a lot of scenarios. Officially encoding this into the model spec would introduce some questions with unclear answers: How common does a value have to be, to get the designation of "default"? How do we decide which inputs get defaults? Are we accidentally misleading users or promoting certain papers/methods by giving extra weight to a "default" value?

All that is to say that I don't really like the term "default", and I might say something like "A common value is 2000 m (citation)" instead. @phargogh do you have any thoughts about defaults?

Reply:
> All that is to say that I don't really like the term "default", and I might say something like "A common value is 2000 m (citation)" instead.

Yes, I agree that "default" implies that the value is a fallback, which isn't quite correct.  I think "recommended" value is maybe closer to what we really mean ... it's more like a "this value should be sufficient for most use cases, but we want you to be able to tweak it if you know what you're doing".

RE: citations, these would be great to include with a recommended value where available!  But this should be able to handle 0+ citations.  A concrete use case for this is that RouteDEM allows the user to provide a band index (recommended value: 1).  DelineateIt has a checkbox marked as True (whether to skip invalid geometry).  No citations needed there.

> There is no representation of a "default value" in the ARGS_SPEC, and I'm not convinced that there should be.

This is true!  We've been shoehorning a default value in to the UI files or just using the parameter set files to represent what might be better represented as default/recommended values.

I have 2 reasons why I think we should seriously consider adding a recommended value to the ARGS_SPEC for non-file inputs:
1. Our science staff sometimes ask for them!  At times the request is better represented as part of a parameter stack, but there do seem to be some 'expert opinion' cases where a natcap-scientist recommended value might be good enough for most.
2. For cases where the model has a recommended or default value anyways (urban cooling example within `execute`), it'd be better to have a single source of truth for that value by referring back to the ARGS_SPEC.  This could also enable us to have a more consistent user experience with recommended/default values across models, too.

If we were to have ARGS_SPEC be the source of truth for recommended values, it would also solve the other concerns you mentioned about values not being available to the workbench and would allow for an individual value to be reset to its original recommended/default value.

Reply:
Great points @emlys , especially about maybe just using a different wording, like "common value" or "recommended starting value" rather than "default".
Specific to this case, please see [this Forums post](https://community.naturalcapitalproject.org/t/urban-cooling-queries-about-parameterization/915) from April 2020 where @royremme wrote,
> "I suggest starting with the 500m value in the appendix that is based on the first studies we have used the model in. The 2000m is an artifact from an older version of the user guide, we need to update that".
That doesn't quite explain my "600m", but I believe this value came from what @newtpatrol and @chrisnootenboom have used most recently for the Twin Cities and what we recommended in the data packet we used for Gretchen's class.

Reply:
>>    "I suggest starting with the 500m value in the appendix that is based on the first studies we have used the model in. The 2000m is an artifact from an older version of the user guide, we need to update that".
>
>That doesn't quite explain my "600m", but I believe this value came from what @newtpatrol and @chrisnootenboom have used most recently for the Twin Cities and what we recommended in the data packet we used for Gretchen's class.

@jagoldstein is it your impression that this value will be specific to the site, or do you think this will be a pretty common value across study areas?  If it's for the sake of demonstration, I'd think this might be better-suited to a parameter set (demonstrating this numeric value in the context of other datasets that made sense for that analysis) than a standalone recommended value.  What do you think about that?

Reply:
@phargogh great points and questions. It is my impression that there can be relatively high uncertainty in the max air blending distance, as described by @chrisnootenboom in [this Forums post](https://community.naturalcapitalproject.org/t/air-temperature-maximum-blending-distance-uncertainty/1795/3). Chris says that 600m comes from "Shatz and Kucharik papers" (not cited in UG). The model was tested in the Twin Cities and Paris, France before release, but _not_ in significantly different locations, like the tropics. I think your idea of a parameter set sounds like it could make more sense than "recommended" or "default" values. I should defer to Chris or Eric though as I am far from an expert on this.

Reply:
@jagoldstein @phargogh I just caught up on this thread and can confirm: the 500m value was derived from regression modeling in the Schatz and Kucharik paper (https://doi.org/10.1175/JAMC-D-14-0107.1) while the 600m value came from similar regression modeling in our golf paper (https://doi.org/10.1016/j.landurbplan.2020.104022). Users should be aware of this limitation when applying the model to other climate zones, but there just aren't many data sources on this value out there. 
It is unclear whether the parameter will be hyper-specific to an individual city--air flow dynamics should be consistent globally, but our current calibration studies in the Twin Cities and Paris result in a range of values from as low as 260m to as high as 1600m. So the jury is definitely out on a single 'default' value even in cities in temperate climates!
That being said, I would reference the two papers I linked above in the users guide if possible, since 500 or 600m is a reasonable first pass for the model.

Reply:
Thanks for this info @chrisnootenboom ! I adjusted the language accordingly and added these two sources to the References section.